src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java

```diff
@@ -277,4 +284,16 @@ public PublicationController(PublicationService publicationService, ValidationSe
             .header(HttpHeaders.CONTENT_DISPOSITION, "attachment; filename=" + fileType)
             .body(file);
     }
+
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "Successfully deleted artefact: {artefactId}"),
+        @ApiResponse(code = 404, message = "No artefact found with the ID: {artefactId}"),
+    })
+    @ApiOperation("Delete a artefact and its list from P&I")
+    @DeleteMapping("/{artefactId}")
+    public ResponseEntity<String> deleteArtefact(@RequestHeader("x-issuer-email") @Email String issuerEmail,
```

Comment:
Is there are reason why we are not using the x-admin to confirm the file can be deleted?

Reply:
the headers here are not used to determine authorisation, a whole security piece is to be played around endpoints that will check if the user can perform certain actions